core/themes/basis/css/skin.css

```diff
@@ -882,6 +882,9 @@ nav.tabs:before {
   /* Background behind primary tabs */
   background-color: #586172;
 }
+.installed-1-30 nav.tabs:before {
```

Comment:
Should we try to match the name of the config setting here? Like `.initial-version-1-30`.

Reply:
The first class added will be the initial version, but all the following classes won't be. It would be weird to have classes like `initial-version-1-30 initial-version-1-31 initial-version-1-32 initial-version-1-33` if someone installs at version 1-33.x. 

I'm open to changing it though -- @stpaultim  recommended `supplemental-1-30` in [this comment ](https://github.com/backdrop/backdrop-issues/issues/4782#issuecomment-2484641807)which I think is better :) 

Naming thing is hard.

Reply:
I think I like `update` even more. Less characters? will push an updated PR.

Reply:
I like the `supplemental` prefix too. To separate the version number more I would add another dash: `supplemental--1-30`.